Automattic/wp-calypso
# My Home: Add Webinar Card & Task

#### Changes proposed in this Pull Request

* Adds a card and task to promote Webinars

#### Testing instructions

In order to make the tasks appear, add the following to `client/my-sites/customer-home/main.jsx`:

`import Webinar from 'my-sites/customer-home/cards/tasks/webinar';`

And then `<Webinar />` on line 104.

In order to make the card appear, add `<Webinars/>` to line 54 in `client/my-sites/customer-home/locations/primary/index.jsx`.

From there, verify that both the tasks and the card work well. The illustration in the card shouldn't be displayed on mobile devices, but there are a further two things to note though:

1. the illustration is temporary as I don't have access to the one in the initial design (cc @danhauk)
2. I'm under the assumption that the logic of when to display them (ie. whether or not the site has launched) shouldn't be added here, but let me know if that's wrong

Also, should the `external` Gridicon be used to indicate that you'll be taken to a screen outside Calypso? 

<img width="1043" alt="Screenshot 2020-04-28 at 16 41 40" src="https://user-images.githubusercontent.com/43215253/80507905-46c65680-896f-11ea-8f3f-b59478f9c19b.png">



Fixes #41339
